.sqlfluff

```diff
@@ -0,0 +1,35 @@
+[sqlfluff]
+verbose = 1
+templater = jinja
+dialect = databricks
+
+# SQLFluff rules: https://docs.sqlfluff.com/en/stable/rules.html
+exclude_rules = RF04,ST06
+
+sql_file_exts = .sql
+max_line_length = 120
+ignore_templated_areas = True
+large_file_skip_byte_limit = 0
```

Comment:
We should introduce a byte limit. This will help to detect files that should be seeds instead of models. 10-20k should be sufficient. Does sqlfluff give any advice on this?

Reply:
Nice catch, I will change to the default of `large_file_skip_byte_limit = 20000`. We can change it later if necessary.